coremain/run.go

```diff
@@ -47,6 +49,11 @@ func Run() {
 	log.SetOutput(os.Stdout)
 	log.SetFlags(LogFlags)
 
+	_, err := maxprocs.Set(maxprocs.Logger(log.Printf))
+	if err != nil {
+		mustLogFatal(err)
```

Comment:
I don't think we should make this fatal.

Reply:
By default, automaxprocs ignore the error in init https://github.com/uber-go/automaxprocs/blob/master/automaxprocs.go#L32.

But since the log in this case contains time, I wanted to align the logs with the current ones in CoreDNS and therefore moved maxprocs.Set to run.go.

What do you prefer? ignore, log or fatal ? :)

Reply:
I think a warning log would be sufficient.

Reply:
Changed to log without fatal.

I added `[WARNING]` before message to understand that this is a warning. The same as in `log` plugin.

Maybe it looks strange, but in CoreDNS we have default go logger. And I think it's not a good idea to use `log` plugin in main.